parley/src/layout/editor.rs

```diff
@@ -111,6 +111,10 @@ where
     BackdeleteWord,
     /// Move the cursor to the cluster boundary nearest this point in the layout.
     MoveToPoint(f32, f32),
+    /// Move the cursor to a byte index.
+    ///
+    /// No-op if index is not a char boundary.
+    MoveToByte(usize),
```

Comment:
For all of these, we should document the affinity semantics (in that it clobbers them entirely).